source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -369,7 +375,7 @@
                         }
                         valuesByItem.hasComments = ko.computed(function () {
                             return valuesByItem.comments.find(function (comment) {
-                                return !!comment.HasComment();
+                                return !!comment.HasComment() && !!comment.isSelfComment;
```

Comment:
`_.contains` does not us predicates, but I understand what you mean. The function for this case is `_.some`
I'll change it